lib/sharingplugininfo.cpp

```diff
@@ -22,19 +22,19 @@
  * Lesser General Public License for more details.
  */
 
-#include "transferplugininfo.h"
+#include "sharingplugininfo.h"
 
-QVariantMap TransferPluginInfo::metaData() const
+QVariantMap SharingPluginInfo::metaData() const
 {
-    return property("_nemo_transferplugininfo_capabilities").toMap();
+    return property("_sailfish_sharingplugininfo_capabilities").toMap();
 }
 
-void TransferPluginInfo::setMetaData(const QVariantMap &metaData)
+void SharingPluginInfo::setMetaData(const QVariantMap &metaData)
 {
-    setProperty("_nemo_transferplugininfo_capabilities", metaData);
+    setProperty("_sailfish_sharingplugininfo_capabilities", metaData);
```

Comment:
Hm, strange things these. Wonder if needed still or possible to do some other way. Related to JB#45043, didn't check the details more now.

Reply:
My guess is that this is also something we had for QML or something but it's not used, anymore anyway. No mentions on device or in any of these repositories. I think we could just remove it.

Reply:
Dropped this whole meta data thingy. I don't think it's used anywhere anymore as we don't support creating accounts within sharing.